### ISSUE TITLE
Missing styles in build ?

### ISSUE THREAD

**Issue by padsbanger** [toptal/simple-react-calendar#215]
Hello,

I have just installed this library using yarn and I tried to include in my app:

` <SimpleReactCalendar activeMonth={new Date()} />`

I am using fresh React boilerplate from create-react-app, no extra configuration. It looks like that styles are missing, but they are also not present in `dist` directory of this package.

Am I missing something ?

**Comment by havenchyk** [comment]
Hey @padsbanger, that's a good question. We didn't deliver any styles by default, instead we relied on classnames from css so you could create your own styles (e.g. from examples folder which we use for storybook), so our calendar is pretty lightweight.

But actually having some default styling is not that bad and probably expected by users

**Comment by padsbanger** [comment]
Yeah, I was looking for a small and simple react calendar too quickly drop into my project while prototyping and I was suprised that there was no default styling.

**Comment by havenchyk** [comment]
So I'd rephrase: we need to deliver some default styling with calendar 😄

**Comment by gugiserman** [comment]
@havenchyk any news on this one? I just found this project and it seemed like a lifesaver for a quick drop-in replacement of our current date range picker, but then noticed it had no default styling and we can't afford the time to style everything from scratch right now 😭
